src/utils.js

```diff
@@ -50,3 +50,7 @@ export const burnItDown = (message?: BurnItDownResponse) => {
     unsubscribe();
   }, timeToBurnItDown);
 };
+
+export const sortByTimestamp = (messages: Array<{timestamp: number}>): Array<Object> => {
```

Comment:
Instead of returning `Array<Object>` I believe you could return `Array<*>` and not lose type inference